packages/contracts/script/deployer/Configured.s.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import { CommonBase } from "forge-std/Base.sol";
+import { console2 } from "forge-std/console2.sol";
+import { StdChains } from "forge-std/StdChains.sol";
+import { stdToml } from "forge-std/StdToml.sol";
+
+/**
+ * @notice Any script that is based upon the multi-network syntax TOML Configuration should extend this.
+ *
+ * @dev This contract extends [CommonBase] to avoid directly extending [Script] or [Test], where it will be used.
+ */
+abstract contract Configured is CommonBase, StdChains {
```

Comment:
This is the codified means of accessing the TOML configuration. There should be no other code that knows about TOML Config Keys.